### ISSUE TITLE
Which package of cityscapes dataset should I use?

### ISSUE THREAD

**Issue by PerryL7s** [krumo/Domain-Adaptive-Faster-RCNN-PyTorch#25]
Hi, thanks for your excellent work. I have never accessed cityscapes dataset before, and the official download page really confuses me since there're so many individual packages (sub-dataset?) like `gtFine`, `gtCoarse`, `leftImg8bit` etc. I'm wondering which package should I download for this project? Are the data structures of these packages actually the same?

Same question comes with foggy_dataset, many thanks :)

**Comment by PerryL7s** [comment]
Does it make sense if I choose `leftImg8bit_trainvaltest` (5000 images) for both cityscapes and foggy_cityscapes dataset?

**Comment by PerryL7s** [comment]
It turns out that `gtFine` is the annotation data for `leftImg8bit`. Guess `gtFine`, `leftImg8bit` & `leftImg8bit_foggy` would work.

**Comment by krumo** [comment]
> It turns out that `gtFine` is the annotation data for `leftImg8bit`. Guess `gtFine`, `leftImg8bit` & `leftImg8bit_foggy` would work.

Thanks for your interest! You are right. `gtFine` contains ground truth labels for training set and validation set in `leftImg8bit` & `leftImg8bit_foggy`. In the Cityscapes to Foggy Cityscapes setting, we use labeled training set of Cityscapes as source domain training data, unlabeled training set of Foggy Cityscapes as target domain training data. The performance on validation set of Foggy Cityscapes is reported to measure the effectiveness of domain adaptation.

**Comment by kxgong** [comment]
> > It turns out that `gtFine` is the annotation data for `leftImg8bit`. Guess `gtFine`, `leftImg8bit` & `leftImg8bit_foggy` would work.
> 
> Thanks for your interest! You are right. `gtFine` contains ground truth labels for training set and validation set in `leftImg8bit` & `leftImg8bit_foggy`. In the Cityscapes to Foggy Cityscapes setting, we use labeled training set of Cityscapes as source domain training data, unlabeled training set of Foggy Cityscapes as target domain training data. The performance on validation set of Foggy Cityscapes is reported to measure the effectiveness of domain adaptation.

Hello, what is the fog factor of foggy cityscapes used for training & testing? 0.005, 0.01 or 0.02, thank you!!!

**Comment by krumo** [comment]
> > > It turns out that `gtFine` is the annotation data for `leftImg8bit`. Guess `gtFine`, `leftImg8bit` & `leftImg8bit_foggy` would work.
> >
> >Thanks for your interest! You are right. `gtFine` contains ground truth labels for training set and validation set in `leftImg8bit` & `leftImg8bit_foggy`. In the Cityscapes to Foggy Cityscapes setting, we use labeled training set of Cityscapes as source domain training data, unlabeled training set of Foggy Cityscapes as target domain training data. The performance on validation set of Foggy Cityscapes is reported to measure the effectiveness of domain adaptation.
>
> Hello, what is the fog factor of foggy cityscapes used for training & testing? 0.005, 0.01 or 0.02, thank you!!!

Hi, 0.02 is used when I perform domain adaptation on foggy cityscapes.

**Comment by kxgong** [comment]
> > > > It turns out that `gtFine` is the annotation data for `leftImg8bit`. Guess `gtFine`, `leftImg8bit` & `leftImg8bit_foggy` would work.
> > > 
> > > 
> > > Thanks for your interest! You are right. `gtFine` contains ground truth labels for training set and validation set in `leftImg8bit` & `leftImg8bit_foggy`. In the Cityscapes to Foggy Cityscapes setting, we use labeled training set of Cityscapes as source domain training data, unlabeled training set of Foggy Cityscapes as target domain training data. The performance on validation set of Foggy Cityscapes is reported to measure the effectiveness of domain adaptation.
> > 
> > 
> > Hello, what is the fog factor of foggy cityscapes used for training & testing? 0.005, 0.01 or 0.02, thank you!!!
> 
> Hi, 0.02 is used when I perform domain adaptation on foggy cityscapes.

Hi, thanks for your quick reply.
